requirements_all.txt

```diff
@@ -1,24 +1,6 @@
 # Home Assistant Core, full dependency set
-
-aiohttp==3.6.1
-astral==1.10.1
-async_timeout==3.0.1
-attrs==19.3.0
-bcrypt==3.1.7
-certifi>=2020.4.5.1
-ciso8601==2.1.3
-importlib-metadata==1.6.0;python_version<'3.8'
-jinja2>=2.11.1
-PyJWT==1.7.1
-cryptography==2.9.2
-pip>=8.0.3
-python-slugify==4.0.0
-pytz>=2020.1
-pyyaml==5.3.1
-requests==2.24.0
-ruamel.yaml==0.15.100
-voluptuous==0.11.7
-voluptuous-serialize==2.4.0
+-c homeassistant/package_constraints.txt
```

Comment:
This change also requires an update to dockerfile.dev, since the COPY command is not using this hierarchy it breaks the dev container rebuild.

Reply:
> This change also requires an update to dockerfile.dev, since the COPY command is not using this hierarchy it breaks the dev container rebuild.

@cmroche, I've updated the `Dockerfile.dev` file in PR #37801